c2corg_ui/views/index.py

```diff
@@ -57,6 +57,18 @@ def following(self):
         return self._get_page(
             'following', 'c2corg_ui:templates/following.html')
 
+    @view_config(route_name='whatsnew',
+                 renderer='c2corg_ui:templates/whatsnew.html')
+    def whatsnew(self):
+        resp, data = call_api(self.settings, 'document/changes')
```

Comment:
I don't understand why you call the API from within a UI server view instead of making a direct request from the browser, as done for the home/profile feeds.